src/Illuminate/Hashing/BcryptHasher.php

```diff
@@ -17,8 +17,8 @@ class BcryptHasher implements HasherContract
     /**
      * Hash the given value.
      *
-     * @param  string  $value
```

Comment:
Please could you revert the cs changes to the phpdoc here, and in the other similar places. :)